### PR TITLE
Add category-theory to ci.

### PR DESCRIPTION
**Pull request by Alizter**
cc @jwiegley

### CODE REVIEW

File: Makefile.ci [hunk 1]
```diff
@@ -14,6 +14,7 @@ CI_TARGETS= \
     ci-bbv \
     ci-bedrock2 \
     ci-bignums \
+    ci-category_theory \
```
Comment (SkySkimmer) [inline Makefile.ci:17]: dependency on equations needs to be added to this makefile too

**Comment by Alizter** [comment]
Forgot to mark as executable. Should be good now.

**Comment by SkySkimmer** [comment]
@coq/ci-maintainers everyone ok with this?

**Comment by Zimmi48** [comment]
OK for me since, according to the author of the library, many regressions could have been detected earlier if we had got it in our CI.

**Comment by Alizter** [comment]
@coq/ci-maintainers anyone willing to assign?

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot: merge now

**Pull request merged**